tests/build_args_test.py

```diff
@@ -0,0 +1,185 @@
+"""For testing the functionality of `BuildArgs` and `convert_build_args_to_argparser`."""
```

Comment:
Create a `python` folder under `tests`, rename this file to `test_build_args.py` (this is the naming convention of python unittest files), and put it under `tests/python/` (because we may have unittests for other language bindings).